README.md

```diff
@@ -1,3 +1,108 @@
+To run a Scala Spark job on AWS you need to compile a [fat jar](https://stackoverflow.com/questions/19150811/what-is-a-fat-jar) that contains your source code for your job and all of the libraries it needs to run.
+This project already has the [sbt-assembly](https://github.com/sbt/sbt-assembly) plugin setup and a [`assemblyMergeStrategy`](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/build.sbt#L13) set up to package the Spark, Hadoop, and AWS SDK together in the fat jar.
+If you need to add in other libraries that do not play well with each other, or you are using a noncompatible version of Spark for this current repo, there are a few good [resources](http://queirozf.com/entries/creating-scala-fat-jars-for-spark-on-sbt-with-sbt-assembly-plugin) available to help you through the needed `build.sbt` changes.
+
+Outside of the needed changes to [SparkJob.scala](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/src/main/scala/SparkJob.scala) and maybe [build.sbt](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/build.sbt), you need to set a few parameters in the [deploy.sh](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/deploy.sh) script. 
+Mainly the [deploymentPath](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/deploy.sh#L2) to your specific S3 bucket, adding a profile to the [AWS CLI command](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/deploy.sh#L14) to upload to your specific S3 bucket if it's private, and changing the resulting fat jar [name](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/deploy.sh#L3) if you please. 
+If you do not have the [AWS CLI](https://aws.amazon.com/cli/) installed and configured you will need to go through the [steps to do that](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) before you can use the `deploy.sh` script. 
+
+## Setting Up The Job On AWS
+
+Once you have your fat jar uploaded to S3 its time to set up the scheduled job on AWS Data Pipeline.
+
+#### Create AWS Data Pipeline
+
+Log on to the AWS dash and make and navigate to the [AWS Data Pipeline](https://console.aws.amazon.com/datapipeline) console click the `Create new pipeline` button.
+
+<img src="https://i.imgur.com/o9omjFr.png" width="800"/>
+
+#### Load The Spark Job Template Definition
+
+Add a name for your pipeline and select the `Import a definition` source option. 
+Included in the [GitHub repo](https://github.com/jessebrizzi/spark-fat-jar-template) for the spark template is a [datapipeline.json](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/datapipeline.json) file that you can import that contains a pre-defined data pipeline for your spark job that should simplify the setup processes.
+The definition contains the node configuration to fire the pipeline off on a schedule and notify you via a [AWS SNS](https://aws.amazon.com/sns/) alarm on a success or failure run.
```

Comment:
Is SNS the only notification mechanism? If not, do you want to mention the other options?

Reply:
The included data pipeline definition is expecting an SNS topic and has the modules set up for it. They could modify the data pipeline to do something else if they wanted, but that would probably be out of scope of the article.